masonite/middleware/MaintenanceModeMiddleware.py

```diff
@@ -0,0 +1,21 @@
+''' Maintainance Mode Middleware '''
+import os
+from config import application
+
+class MaintenanceModeMiddleware:
+    ''' Middleware To Check Maintaince Mode'''
+
+    def __init__(self, Request:
+        ''' Inject Any Dependencies From The Service Container '''
+        self.request = Request
+
+
+    def before(self):
+        ''' Run This Middleware Before The Route Executes '''
+        down = os.path.exists(os.path.join(application.BASE_DIRECTORY, 'bootstrap/down'))
+        if down is True:
+            self.request.status('503 Service Unavailable')
+            return # todo return response request lifecycle should stop 
```

Comment:
@josephmancuso 

I got stuck what to return on this line.

Reply:
may not have to return anything if that works. Middleware is really just designed to changed some configurations before reaching (or not reaching) the view

Reply:
but do use double quotes now instead of single quotes. 2.1 will be more closer to "PEP 8" standards than 2.0 was

Reply:
Not working as expected, I was testing with the freshly installed project via `craft new hello` and it was always showing that welcome page even after this middleware is implemented.